kolibri/plugins/setup_wizard/assets/src/views/ImportMultipleUsers.vue

```diff
@@ -91,6 +92,8 @@
         isPolling: false,
         // array of user/learner ids
         learnersBeingImported: [],
+        // To be removed
+        isImportingUserDisabled: false,
```

Comment:
A thought just occurred during review. I think that we might want to instead just redirect the user to the LoadingTasksPage once they start an import from the Multiple Users list. I think this will improve the UX as currently it just feels rude to disable every other button on the screen.

Additionally, when you click to "Go back" -- you are landing back on that page w/ the user credentials form. 

I think that on the router, adding a navigation guard to check something like "is a user being imported right now? OR has a user been successfully imported?" -- then if those things are true, we direct the user to the LoadingTaskPage.